src/tqec/compile/tree/annotators/detectors.py

```diff
@@ -164,6 +164,7 @@ def __init__(
         detector_database: DetectorDatabase | None = None,
         only_use_database: bool = False,
         lookback: int = 2,
+        parallel: bool = False,
```

Comment:
Same comment as above, maybe change that to be an `int` to give more freedom to the user?